Web/Web.config

```diff
@@ -58,7 +58,7 @@
     <add key="environment" value="local" />
     <add key="config:EnableTiles" value="1" />
     <add key="config:EnableLoader" value="1" />
-    <add key="config:Project" value="ElmahBucket" />
+    <add key="config:Project" value="Monitorr" />
```

Comment:
There is also database name for hangfire(line 69) <add key="hangfire:DatabaseName" value="EbHangfire" />.

Reply:
key="hangfire:DatabaseName